README.md

```diff
@@ -96,34 +110,38 @@ cd SmartSim-Scaling
 pip install -r requirements.txt
 ```
 
+> Note that if you are using a Cray machine, you will need to run `CC=cc CXX=CC pip install -r requirements.txt`.
+
 Lastly, the C++ applications themselves need to be built. To complete this, 
-one CMake edit is required. Navigate to your respective tests `CMakeLists.txt` file.
-If you need assistance finding the file path, please reference the chart below.
-Near the top of `CMakeLists.txt`, on line
-`set(SMARTREDIS "../../SmartRedis" CACHE PATH "Path to SmartRedis root directory")`, 
-change the path to the ``SMARTREDIS`` variable to
-the top level of the directory where you built or installed the SmartRedis library. 
+one CMake edit is required. When running `cmake ..`, 
+change the path to the ``SMARTREDIS`` variable to the top level of the directory 
+where you built or installed the SmartRedis library using the ``-DSMARTREDIS`` flag.
+An example of this is shown below.
+
+All tests can be built by running
+
+```bash
+  cd <language name>-<test name> # ex. cpp-inference for the cpp inference tests
+  mkdir build && cd build
+  cmake .. -DSMARTREDIS=/path/to/SmartRedis
+  make
+```
+
```

Comment:
Can you make a ticket for us to implement a makefile to help users automate the building of each test app?

Reply:
yes